src/ssl.c

```diff
@@ -15187,9 +15188,10 @@ WOLFSSL_X509* wolfSSL_X509_load_certificate_buffer(
                 x509 = (WOLFSSL_X509*)XMALLOC(sizeof(WOLFSSL_X509), NULL,
                                                              DYNAMIC_TYPE_X509);
                 if (x509 != NULL) {
+                    XMEMSET(x509, 0, sizeof(WOLFSSL_X509));
```

Comment:
Please remove the 3 xmemset's on the x509 structure unless absolutely necessary? The first thing InitX509 does is xmemset the structure to 0.

Reply:
Agreed.